package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-querybuilder",
-  "version": "3.7.0",
+  "version": "3.7.1",
```

Comment:
Don't bump the version yet.  We do that when we publish.

Reply:
ok

Reply:
Updated. 👍